spec/rspec/rails/matchers/have_http_status_spec.rb

```diff
@@ -382,12 +382,12 @@ def create_response(opts = {})
     it_behaves_like "supports different response instances" do
       subject(:matcher) { have_redirect_status }
 
-      let(:code) { 333 }
+      let(:code) { 308 }
     end
 
     describe "matching a response" do
       it "returns true for a response with a 3xx status code" do
-        any_3xx_code = 333
+        any_3xx_code = 308
```

Comment:
The reason I made this change is that somewhere along the way, rails 5 no-longer accepts 3xx codes that aren't part of the spec in a bunch of places. 333 is not part of the spec. Given that all old versions of rails support any http response code that is in the spec, this seemed like the easiest change to make for this PR to work.